GooglePlayInstant/Editor/QuickDeploy/QuickDeployWindow.cs

```diff
@@ -155,26 +151,35 @@ private void OnGuiCreateBundleSelect()
 
             if (GUILayout.Button("Build AssetBundle"))
             {
+                // TODO: Change UI to have default path and a browse button, and avoid prompting user every time they want to build.
+                var assetBundleBuildPath = EditorUtility.SaveFilePanel("Save AssetBundle", "", "quickDeployAssetBundle", "");
+                // Do nothing if user cancelled.
+                if (string.IsNullOrEmpty(assetBundleBuildPath))
+                {
+                    return;
+                }
+
+                QuickDeployConfig.AssetBundleFileName = assetBundleBuildPath;
+                QuickDeployConfig.SaveConfiguration(ToolBarSelectedButton.CreateBundle);
                 try
                 {
-                    QuickDeployConfig.AssetBundleFileName = EditorUtility.SaveFilePanel("Save AssetBundle", "", "", "");
-                    // TODO(audace): build the assetbundles
+                    AssetBundleBuilder.BuildQuickDeployAssetBundle(GetEnabledSceneItemPaths());
                 }
                 catch (Exception ex)
                 {
                     DialogHelper.DisplayMessage(AssetBundleBuildErrorTitle,
                         ex.Message);
                     throw;
                 }
+                
+                EditorGUIUtility.ExitGUI();
```

Comment:
The problem Audace was trying to fix was one where the save file panels, when nexted in between BeginHorizontal(s), it would sometimes throw errors. (Most notably on a mac devicem when switching between pages.)

Reply:
Removing it since it is handled in Jess' PR